tests/conftest.py

```diff
@@ -246,3 +248,24 @@ def telescope_north_test_file():
 @pytest.fixture
 def telescope_south_test_file():
     return "data/layout/telescope_positions-South-TestLayout.ecsv"
+
+
+@pytest.fixture
+def corsika_output_file_name_string():
+    return "tests/resources/tel_output_10GeV-2-gamma-20deg-CTAO-South"
+
+
+@pytest.fixture
+def corsika_output_file_name(io_handler, corsika_output_file_name_string):
+    corsika_output = gen.find_file(
+        corsika_output_file_name_string,
+        io_handler.get_output_directory(dir_type="corsika_output", test=True),
+    )
+    return corsika_output
+
+
+@pytest.fixture
+def corsika_output_instance_set_histograms(db, io_handler, corsika_output_file_name):
```

Comment:
Since you don't have another instance, maybe we don't need the `set_histograms` in the name?

Reply:
we do have another instance which is used only in `test_corsika_output` file. one instance is for applying the set_histograms while the other stays fixed. this can be confusing because the behaviour might change if we set the histograms in a different way. Therefore, I created one instance for the case in which I need to set the histograms (and then I always set them) and another one for the cases in which we do not need to set them

Reply:
OK this is because I didn't look at `test_corsika_output` yet. Let me keep this unresolved for now and look at it first then.